### ISSUE TITLE
[BUG] Vehicle bar doesn't open/work at all when in combat

### ISSUE THREAD

**Issue by TommiMa** [Karl-HeinzSchneider/WoW-DragonflightUI#173]
**Describe the bug**
Hi, pretty much the vehicle bar doesn't open when being in combat and it completely messes up with your other bars aswell. For example on Magmaw when player needs to click on the spike or on Nerfarian once you get Mindcontrolled. In additional context I have a tiny clip what it does on Nefarian when getting mindcontrolled. I managed to replicate this bug outside of raid aswell in the Darkmoon Faire where I put myself in the combat and then enter one of the minigames, 100% of the time it bugs out similarly. After disabling the Action bar module everything is back to normal and it stops bugging out so it has something to do with that specific module. (Streamable links in additional context)

**Error Log**
None.

**Versions :**
 - WoW: Cata Classic
 - Addon v0.13.2

**Additional context**
https://gyazo.com/9f1a90e9c5598edf58590d7cf75159a9
https://streamable.com/ddysoo (Action bar module enabled)
https://streamable.com/50b7d0 (Action bar module disabled)

**Comment by MannySJH** [comment]
Getting the same thing today on nefarian

**Comment by Karl-HeinzSchneider** [comment]
Hey,
is this new behaviour, or was it like that every id? I'm not really playing Cata right now, only developing the addon. I know in wotlk everything was/is fine, so maybe they changed something, or its a different mechanic. 
The videos are really good! Nice find with the darkmoon faire; I'll take a look.

**Comment by TommiMa** [comment]
It is a new behaviour with this addon, everything was fine until I updated the addon to 0.13.2 version on Monday the 29th. There was actually a similar/same bug back in classic wotlk on Malygos fight with the dragon "vehicle" in last phase which was caused by Modern Target Frame addon. https://www.reddit.com/r/classicwow/comments/xz17jc/malygos_phase_3_action_bar_bug_is_it_just_me/

**Comment by Karl-HeinzSchneider** [comment]
I think I found the issue:
In one of the last updates, I changed the logic to show/hide actionbars, because the default Bars 4+5 are dependend on each other, so you cant activate 5 if you dont have 4 activated. I want to change that to make editing easier, but there isnt a good way to force activate every extra actionbar (and let DragonflightUI handle the visibility).
My method tainted the blizzard code, so I deactivated that with 8235553284f44c450c6ae0a892bd757bdf9ba8b0.

I'll work on this at a later point, and might add some ingame messsage or something. But for now, it should work again.
